src/variables/sss.ts

```diff
@@ -3,4 +3,5 @@ import {SssProviders} from '@app/services/provider-sss';
 export const ProviderNameMap = {
   [SssProviders.apple]: 'iCloud',
   [SssProviders.google]: '\nGoogle Drive',
+  [SssProviders.custom]: 'iCloud or Google Drive',
```

Comment:
I think we should place newline at the beginning and add small adjustment to the name
```suggestion
  [SssProviders.custom]: '\nHAQQ Provider',
```